components/suggestion/command_provider/command_provider.tsx

```diff
@@ -198,24 +194,17 @@ export default class CommandProvider extends Provider {
                     component: CommandSuggestion,
                 });
             },
-        ).catch(
```

Comment:
Not catching this may result in any future problem? I am mainly thinking of "uncatched exception" messages on the console, but maybe something bigger could happen.

Reply:
@larkox This function is not `await`'d or `then`'d, so in that regard there is no issue with an exception occurring. 1/5 it's better for an exception to show in the console than to swallow and ignore. Empty catch blocks cause more issues than they prevent.